addons/mail/tests/test_link_preview.py

```diff
@@ -121,19 +121,25 @@ def test_link_preview(self):
                 {'source_url': self.source_url, 'message_id': self.existing_message.id}
                 for _ in range(throttle)
             ])
-            message = self.test_partner.message_post(
-                body=Markup(f'<a href={self.source_url}>Nothing link</a>'),
-            )
-            self._reset_bus()
-
-            def get_bus_params():
-                return (
-                    [(self.cr.dbname, "res.partner", self.env.user.partner_id.id)],
-                    [
-                        {
-                            "type": "mail.record/insert",
-                            "payload": {
-                                "LinkPreview": [
+
+            payloads = [
+                f"<a href={self.source_url}>Nothing link</a>",
+                f"[Nothing link]({self.source_url})"
+            ]
+            for link in payloads:
+                message = self.test_partner.message_post(
+                    body=Markup(link),
+                )
```

Comment:
Could we have the 2 links on the same message so we don't need to loop and change indent?

Reply:
I guess it works too :)